_episodes/02-licensing.md

```diff
@@ -1,17 +1,19 @@
 ---
 title: "Add a License"
-teaching: 0
-exercises: 0
+teaching: 15
+exercises: 15
 questions:
 - "Why is a license important?"
 - "When is a license important?"
 - "How to choose a license?"
+- "How does my license interact with a library and its license"
```

Comment:
What library? For the readers, it may not be clear, that we are talking about the external, libraries, packages, modules etc.